Sources/MapFilter/ValueSliderControl.swift

```diff
@@ -0,0 +1,305 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+typealias SliderValueKind = Comparable & Numeric
+
+protocol ValueSliderControlDelegate: AnyObject {
+    func valueSliderControl<ValueKind: SliderValueKind>(_ valueSliderControl: ValueSliderControl<ValueKind>, didChangeValue: StepValue<ValueKind>)
+}
+
+class ValueSliderControl<ValueKind: SliderValueKind>: UIControl {
```

Comment:
This doesn't have to be a UIControl does it?

Reply:
Changed to a view 👍